index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine">
+    <title>MEdicore</title>
+    <link href="./styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="section--pink">
+        <div class="menu-bar">
+            <div class="menu-bar__section menu-bar__section--text-big">
+                <div class="menu-bar__link">Medical Core</div>
+            </div>
+            <div class="menu-bar__section menu-bar__section--text-muted menu-bar__links">
+                <div class="menu-bar__link">Home</div>
+                <div class="menu-bar__link">Product</div>
+                <div class="menu-bar__link">Pricing</div>
+                <div class="menu-bar__link">Contact</div>
+            </div>
+            <div class="menu-bar__section">
+                <button class="button button--white">Login</button>
+                <button class="button button--uppercase button--blue">join us <img src="./images/Vector.png"> </button>
+            </div>
+        </div>
+        <div class="welcome">
+            <div class="welcome__content">
+                <label class="welcome__text--blue welcome__text--big-in-sm">Welcome</label>
+                <label class="welcome__text--big">A Great Place to Receive Care</label>
+                <label class="welcome__text--muted">Get your best looking smile now!</label>
```

Comment:
For the labels, you are using modifiers without specifying the elements' classes. The best practice with BEM is to use a block or an element class alongside their respective modifiers. For example: `<button class="search-form__button search-form__button-disabled">Search</button>` [Click here](https://getbem.com/naming/) for more info.

Reply:
Confused on how to resolve this